src/main/webapp/WEB-INF/jsp/signup.jsp

```diff
@@ -25,8 +23,52 @@ label {
 }
 </style>
 
+<script type="text/javascript">
+
+   //Created / Generates the captcha function    
+    function DrawCaptcha()
+    {
+    	document.getElementById("submitbutton").disabled = true;
+        var a = Math.ceil(Math.random() * 10)+ '';
+        var b = Math.ceil(Math.random() * 10)+ '';       
+        var c = Math.ceil(Math.random() * 10)+ '';  
+        var d = Math.ceil(Math.random() * 10)+ '';  
+        var e = Math.ceil(Math.random() * 10)+ '';  
+        var f = Math.ceil(Math.random() * 10)+ '';  
+        var g = Math.ceil(Math.random() * 10)+ '';  
+        var code = a + ' ' + b + ' ' + ' ' + c + ' ' + d + ' ' + e + ' '+ f + ' ' + g;
+        document.getElementById("txtCaptcha").value = code;
+        document.getElementById("txtInput").value = '';
+    }
+
+    // Validate the Entered input aganist the generated security code function   
+    	
+	function ValidCaptcha(){

```

Comment:
Is this disabling the submit button on page loaded? looks like when he click on validate and if validation fail then only it will disable submit button else it is enabled always i didnt test those but looking at code you are not disabling on load so just asking.